rpmbuild.c

```diff
@@ -273,20 +287,6 @@ static struct poptOption optionsTable[] = {
    POPT_TABLEEND
 };
 
-static int checkSpec(rpmts ts, rpmSpec spec)
-{
-    int rc;
-    rpmps ps = rpmSpecCheckDeps(ts, spec);
-
-    if (ps) {
-	rpmlog(RPMLOG_ERR, _("Failed build dependencies:\n"));
-	rpmpsPrint(NULL, ps);
-    }
-    rc = (ps != NULL);
-    rpmpsFree(ps);
-    return rc;
-}
-
```

Comment:
Please split this move of build-requires checking into librpmbuild with a new special return code to a separate commit. It makes for a nice independent and logical step that serves as a regression checkpoint against existing behavior and also makes the final commit of the actual new feature more contained.

Reply:
BTW that split also will also give a nice logical place to put the python pass-the-ts changes into.

Reply:
Still kinda hesitant to just lump those patches together. But the move of the build requires check is a separate patch now.

Reply:
Works for me.